cmd/wksctl/addon/build/build.go

```diff
@@ -25,9 +25,9 @@ var addonBuildOptions struct {
 }
 
 func init() {
-	Cmd.PersistentFlags().StringVarP(&addonBuildOptions.outputDirectory, "output-directory", "o", "", "manifest output directory")
-	Cmd.PersistentFlags().StringVarP(&addonBuildOptions.imageRepository, "image-repository", "r", "", "use this container repository for addon images")
-	Cmd.PersistentFlags().StringArrayVarP(&addonBuildOptions.params, "params", "p", nil, "addon input parameters e.g. --params foo=bar --params baz=qux")
+	Cmd.Flags().StringVarP(&addonBuildOptions.outputDirectory, "output-directory", "o", "", "manifest output directory")
```

Comment:
this is, technically speaking, a logical change, isn't it?  It makes sense that it should not be necessary to make it a persistent flag considering it has no further sub commands, but it'd be nice, perhaps, if you could provide some context as to why this PR is the right PR to make this change.  That goes for all of the other places this was changed, I believe.

Reply:
The spirit of this PR (and the rest of #33 so far) is about organizing our commands better - by separating them so that unwanted side effects and interactions are limited - which enables us to provide them as Go APIs.
Authors' intentions were clearly to define these flags for _this_ particular command and nothing else, so fixing `PersistentFlags` to limit these side effects feels in line with the big change to me.

Reply:
Updated the PR description to reflect `PersistentFlags`->`Flags`

Reply:
cool.  that makes sense.  all I was trying to say is that switching this up is itself a side effect, but I agree with your thinking and support the (quite minor) change.  In fact, we might be talking about no practical change _at this time_ if there are not sub commands of the other commands, but I didn't look into them all individually.  good stuff! thanks!